lte/gateway/python/integ_tests/s1aptests/test_attach_ics_drop_with_mme_restart.py

```diff
@@ -110,7 +110,8 @@ def test_attach_ics_drop_with_mme_restart(self):
         )
 
         print("******************** Restarting MME service on gateway")
-        self._s1ap_wrapper.magmad_util.restart_services(["mme"])
+        wait_for_restart = 30
+        self._s1ap_wrapper.magmad_util.restart_services(["mme"], wait_for_restart)
 
```

Comment:
Thanks for catching that. There may be a similar change necessary in `s1aptests/test_attach_ics_failure_with_mme_restart.py` as well.

Reply:
Thanks, I changed it there as well.